tests/test_geoutils.py

```diff
@@ -13,9 +13,7 @@
 
 @pytest.fixture()
 def path_data():
-    path_module = os.path.dirname(os.path.dirname(os.path.abspath(inspect.getsourcefile(gr))))
-    print(path_module)
-    data_folder = os.path.join(path_module, 'tests', 'data')
+    data_folder = os.path.join('tests', 'data')
```

Comment:
Are you sure this will work from anywhere? Don't you need to specify the module path?

Reply:
Ok, I'll accept the PR and CI should tell us if that works anyway ;-)

Reply:
The previous approach, using abspath, failed on CI - 'file not found' errors. This approach works - see the tick next to #69cd427.

Reply:
(and locally, simply run `pytest` from the dev folder and it will automatically sniff out the tests

Reply:
But what if we want to use this data from somewhere else, let's say a notebook folder with some examples. Will that still work?

Reply:
I haven't thought about that side of it - to do so, one would need to import test_geoutils then run the function, which seems a bit odd. perhaps we need to look at how other big packages manage their demo data instead, so that we are not relying on the test suite to provide this.

Reply:
opened #70

Reply:
yes, this is why I wrote it this way in the first place! it was a pain to run part of tests from a console, like through PyCharm, with a relative path while the previous approach worked from both command line and Python project console (based on a previous comment from Amaury, I think); shame it didn't work for CI